cdap-gateway/src/main/java/co/cask/cdap/gateway/router/handlers/SecurityAuthenticationHttpHandler.java

```diff
@@ -80,6 +83,23 @@ public SecurityAuthenticationHttpHandler(String realm, TokenValidator tokenValid
     this.discoveryServiceClient = discoveryServiceClient;
     this.discoverables = discoveryServiceClient.discover(Constants.Service.EXTERNAL_AUTHENTICATION);
     this.configuration = configuration;
+    this.bypassPattern = createMatcher(configuration.get(Constants.Security.Router.BYPASS_AUTHENTICATION_REGEX));
```

Comment:
I don't have much to add to this, sorry. If this is more of an operational property (from JIRA), I think cConf is a fine place to put this property. @poornachandra @dereklwood Any thoughts?